### PR TITLE
Ds responding reviews

### PR DESCRIPTION
**Pull request by DSuveges**
Based on the reviewer's comments I have made a few changes:

* I changed *"coordinates"* to *"location"*  on the gene page and on the association table.
* I have changed the Ensembl data sources to ensure we are extracting data from the same Ensembl version that the mapping is done. 
* Change the solr search part to make sure we are showing the same data on the search results as on the gene page.

### CODE REVIEW

File: goci-interfaces/goci-ui/src/main/resources/static/js/resultexpansion.js [hunk 1]
```diff
@@ -330,6 +330,9 @@ function addResults(data, expand, id) {
                         var geneDescription = '' // initializing empty description
 
                         var descriptionElements = descriptionTruncated.split("|");
+                        if (! descriptionElements[0]){
+                            descriptionElements[0] = "No description available.";
+                        }
```
Comment (twhetzel) [inline goci-interfaces/goci-ui/src/main/resources/static/js/resultexpansion.js]: In the future, will this default text be added to the Solr document directly?

File: goci-interfaces/goci-ui/src/main/resources/static/js/solrsearch.js [hunk 2]
```diff
@@ -339,6 +339,9 @@ function processData(data) {
                     var variantDescription = '' // initializing empty description
 
                     var descriptionElements = descriptionTruncated.split("|");
+                    if (! descriptionElements[0]){
+                        descriptionElements[0] = "No description available.";
+                    }
```
Comment (twhetzel) [inline goci-interfaces/goci-ui/src/main/resources/static/js/solrsearch.js]: In the future, will this default text be added to the Solr document directly?

Comment (DSuveges) [reply goci-interfaces/goci-ui/src/main/resources/static/js/solrsearch.js]: Yes.

File: goci-interfaces/goci-ui/src/main/resources/static/js/resultexpansion.js (hunk 1 above)
Comment (DSuveges) [reply goci-interfaces/goci-ui/src/main/resources/static/js/resultexpansion.js]: Yes.

**Comment by DSuveges** [comment]
As the last commit only contained really minor modifications of the tooltips, I'm merging the branch.

**Pull request merged**